sdks/python/apache_beam/ml/rag/enrichment/bigquery_vector_search_it_test.py

```diff
@@ -0,0 +1,714 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import logging
+import secrets
+import time
+import unittest
+
+import apache_beam as beam
+from apache_beam.io.gcp.bigquery_tools import BigQueryWrapper
+from apache_beam.io.gcp.internal.clients import bigquery
+from apache_beam.ml.rag.enrichment.bigquery_vector_search import BigQueryVectorSearchEnrichmentHandler
+from apache_beam.ml.rag.enrichment.bigquery_vector_search import BigQueryVectorSearchParameters
+from apache_beam.ml.rag.types import Chunk
+from apache_beam.ml.rag.types import Content
+from apache_beam.ml.rag.types import Embedding
+from apache_beam.testing.test_pipeline import TestPipeline
+from apache_beam.testing.util import assert_that
+from apache_beam.testing.util import equal_to
+from apache_beam.transforms.enrichment import Enrichment
+
+try:
+  from google.api_core.exceptions import BadRequest
```

Comment:
This try block needs to include your other BQ imports, otherwise they will throw when they look for gcp deps:

```
_ ERROR collecting apache_beam/ml/rag/enrichment/bigquery_vector_search_it_test.py _
ImportError while importing test module '/Users/runner/work/beam/beam/sdks/python/apache_beam/ml/rag/enrichment/bigquery_vector_search_it_test.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
apache_beam/ml/rag/enrichment/bigquery_vector_search_it_test.py:26: in <module>
    from apache_beam.ml.rag.enrichment.bigquery_vector_search import BigQueryVectorSearchEnrichmentHandler
target/.tox/py39-macos/lib/python3.9/site-packages/apache_beam/ml/rag/enrichment/bigquery_vector_search.py:26: in <module>
    from google.cloud import bigquery
E   ModuleNotFoundError: No module named 'google.cloud'
```

Reply:
This is causing the (non-flake) presubmit failures